test/e2e_node/jenkins/e2e-node-jenkins.sh

```diff
@@ -28,7 +28,16 @@ set -x
 
 . $1
 
+if [ "$INSTALL_GODEP" = true ] ; then
```

Comment:
Please do all this in a rule in a makefile, so humans can call as much or as little of the build process as they want.